src/components/Main/DiscordInvite/DiscordInvite.tsx

```diff
@@ -1,15 +1,18 @@
 import React from 'react';
-
+import {Row, Col} from 'react-bootstrap';
+// import './discordstyle.css';
 // TODO: Add text and link
 // TODO: Add graphic at the end of the page
 
 function DiscordInvite() {
   return (
-    <div>
-      <h4>Want to join Girls Who Code at UVA?</h4>
-      <a href="#" target="_blank"><h2>Join our Discord server (arrow)</h2></a>
-      <img src="#" alt="Orange wave saying 'Get Started'" />
-    </div>
+    <Row>
+      <Col sm={12}>
+        <h4 className="text-turq join"><b>Want to join Girls Who Code at UVA?</b></h4>
```

Comment:
there shouldn't be a need for `<b>`, make sure you merge with main to get the most up to date styling